models/get.models.js

```diff
@@ -13,3 +13,13 @@ exports.fetchApiEndpoints = () => {
         return JSON.parse(endpointsDescription);
     })
 }
+
+exports.fetchArticleById = (articleId) => {
+    return db.query('SELECT * FROM articles WHERE article_id = $1', [articleId])
+    .then((result) => {
+        if(result.rows.length === 0){
+           return Promise.reject("Not found")
```

Comment:
I would reject this with an object with both status and message, as opposed to just a string, to make handling your errors in your middleware easier